test/geometry/transform/test_affine.py

```diff
@@ -14,46 +14,161 @@ def test_smoke(self, device, dtype):
         out = kornia.resize(inp, (3, 4))
         assert_allclose(inp, out, atol=1e-4, rtol=1e-4)
 
-    def test_no_batch(self, device, dtype):
+        # 2D
+        inp = torch.rand(3, 4, device=device, dtype=dtype)
+        out = kornia.resize(inp, (3, 4))
+        assert_allclose(inp, out, atol=1e-4, rtol=1e-4)
+
+        # 3D
         inp = torch.rand(3, 3, 4, device=device, dtype=dtype)
-        out = kornia.resize(inp, (2, 5))
-        assert out.shape == (3, 2, 5)
+        out = kornia.resize(inp, (3, 4))
+        assert_allclose(inp, out, atol=1e-4, rtol=1e-4)
+
+        # arbitrary dim
+        inp = torch.rand(1, 2, 3, 2, 1, 3, 3, 4, device=device, dtype=dtype)
+        out = kornia.resize(inp, (3, 4))
+        assert_allclose(inp, out, atol=1e-4, rtol=1e-4)
 
     def test_upsize(self, device, dtype):
         inp = torch.rand(1, 3, 3, 4, device=device, dtype=dtype)
         out = kornia.resize(inp, (6, 8))
         assert out.shape == (1, 3, 6, 8)
 
+        # 2D
+        inp = torch.rand(3, 4, device=device, dtype=dtype)
```

Comment:
fro next PR - different shape should be addressed via `pytest.mark.parametrize`

Reply:
Got it!